Not able to go get Dgraph repo in Darwin

### What version of Dgraph are you using?
Master

### Steps to reproduce the issue (command/config used to run Dgraph).

```
GO111MODULE=on go get -v github.com/dgraph-io/dgraph/dgraph

#OR
go get -v github.com/dgraph-io/dgraph/dgraph
GO111MODULE=off go get -v github.com/dgraph-io/dgraph/dgraph
```


### Expected behaviour and actual result.

```
GO111MODULE=on go get -v github.com/dgraph-io/dgraph/dgraph
```
### Result

```
(...)
google.golang.org/grpc/internal/transport
google.golang.org/grpc
google.golang.org/grpc/health/grpc_health_v1
github.com/dgraph-io/dgo/protos/api
google.golang.org/grpc/health
github.com/dgraph-io/dgo
github.com/dgraph-io/dgraph/protos/pb
# github.com/dgraph-io/dgraph/protos/pb
../../../go/pkg/mod/github.com/dgraph-io/dgraph@v1.1.0/protos/pb/pb.pb.go:7132:29: m.Kv[iNdEx].MarshalToSizedBuffer undefined (type *pb.KV has no field or method MarshalToSizedBuffer)
../../../go/pkg/mod/github.com/dgraph-io/dgraph@v1.1.0/protos/pb/pb.pb.go:7195:29: m.Kv[iNdEx].MarshalToSizedBuffer undefined (type *pb.KV has no field or method MarshalToSizedBuffer)
```
### And

```
GO111MODULE=off go get -v github.com/dgraph-io/dgraph/dgraph
#OR
go get -v github.com/dgraph-io/dgraph/dgraph
```

```
github.com/dgraph-io/dgraph/conn
# github.com/dgraph-io/dgraph/conn
../../../go/src/github.com/dgraph-io/dgraph/conn/node.go:140:16: not enough arguments in call to n.Applied.Init
	have (nil)
	want (*y.Closer, bool)
../../../go/src/github.com/dgraph-io/dgraph/conn/raft_server.go:164:35: confState.Nodes undefined (type *raftpb.ConfState has no field or method Nodes)
```

--- a/README.md
+++ b/README.md
@@ -43,10 +43,21 @@ docker pull dgraph/dgraph:latest
 
 ## Install from Source
 
-If you want to install from source, you can use `go get` to install to `$GOPATH/bin`.
+If you want to install from source, install Go 1.11+ or later and the following dependencies:
+
+Ubuntu:
+```bash
+sudo apt-get update
+sudo apt-get install gcc make
+```
+
+Then clone the Dgraph repository and use `make install` to install the Dgraph binary to `$GOPATH/bin`.
+
 
 ```bash
-go get -v github.com/dgraph-io/dgraph/dgraph
+git clone https://github.com/dgraph-io/dgraph.git
+cd ./dgraph
+make install
 ```
 
 ## Get Started

--- a/wiki/content/deploy/index.md
+++ b/wiki/content/deploy/index.md
@@ -49,17 +49,27 @@ dgraph
 #### Building from Source
 
 {{% notice "note" %}}
-Ratel UI is closed source right now, so you cannot build it from source. But you can connect to your Dgraph instance
-through Ratel UI installed using any of the methods listed above.
+You can build the Ratel UI from source seperately following its build
+[instructions](https://github.com/dgraph-io/ratel/blob/master/INSTRUCTIONS.md).
+Ratel UI is distributed via Dgraph releases using any of the download methods
+listed above.
 {{% /notice %}}
 
-Make sure you have [Go](https://golang.org/dl/) (version >= 1.8) installed.
+Make sure you have [Go](https://golang.org/dl/) v1.11+ installed.
+
+You'll need the following dependencies to install Dgraph using `make`:
+```bash
+sudo apt-get update
+sudo apt-get install gcc make
+```
 
 After installing Go, run
 ```sh
 # This should install dgraph binary in your $GOPATH/bin.
 
-go get -v github.com/dgraph-io/dgraph/dgraph
+git clone https://github.com/dgraph-io/dgraph.git
+cd ./dgraph
+make install
 ```
 
 If you get errors related to `grpc` while building them, your

